cli/Makefile

```diff
@@ -30,6 +30,9 @@ GENERATED_FILES = internal/turbodprotocol/turbod.pb.go internal/turbodprotocol/t
 turbo: go-turbo$(EXT)
 	cargo build --manifest-path ../crates/turborepo/Cargo.toml
 
+turbo-prod: go-turbo$(EXT)
```

Comment:
@NicholasLYang @chris-olszewski @arlyon did you have any other plans to enable this?

Reply:
Not really. The actual release process just directly invokes `cargo` with `--release`. The only concern I have is that we release with `rustls-tls` even though `native-tls` is activated for development workflows. We should probably just switch to `rustls-tls` by changing the default feature in `crates/turborepo/Cargo.toml`. That way we're benchmarking with the correct setup.

Reply:
Is this good enough? https://github.com/vercel/turbo/pull/4239